js/app/config.js

```diff
@@ -56,7 +59,10 @@ define(function(){
                 decayRate : 0.95,
                 minThreshold : 60
             }
-        }
+        },
+        points : [ 1.5, 2, 2.5, 3 ],
+        checkpoints : [ 100, 500, 5000 ],
+        speeds : [ 4, 3, 2, 1 ]
```

Comment:
haven't made it past level 3, so I wouldn't know.  Also, the checkpoints, points, and speeds aren't really important.  I figure that we can tweak them.